lib/oneview-sdk/resource/api200/firmware_bundle.rb

```diff
@@ -22,7 +22,7 @@ class FirmwareBundle
       # @param [OneviewSDK::Client] client The client object for the OneView appliance
       # @param [String] file_path
       # @param [Integer] timeout The number of seconds to wait for completing the request
-      # @return [OneviewSDK::FirmwareDriver] if the upload was sucessful, return a FirmwareDriver object
+      # @return [OneviewSDK::FirmwareDriver] if the upload was successful, return a FirmwareDriver object
       def self.add(client, file_path, timeout = READ_TIMEOUT)
```

Comment:
Do we want to call the new client#upload_file method here? (Same for [golden_image.rb](https://github.com/HewlettPackard/oneview-sdk-ruby/blob/master/lib/oneview-sdk/image-streamer/resource/api300/golden_image.rb))